lib/config/discover.ml

```diff
@@ -1,21 +1,28 @@
-open Base
-open Stdio
 module C = Configurator.V1
+module S = String
 
-let write_sexp fn sexp =
-  Out_channel.write_all fn ~data:(Sexp.to_string sexp)
+let cfig c =
+  let default: C.Pkg_config.package_conf = {
+      libs = ["-lR"; "-lRmath"];
+      cflags = [] }
+  in
+  let rlib = C.Process.run c "pkg-config" ["--libs-only-L"; "--libs-only-l"; "libR"] in
+  let rmlib = C.Process.run c "pkg-config" ["--libs-only-L"; "--libs-only-l"; "libRmath"] in
+  let rflg = C.Process.run c "pkg-config" ["--cflags"; "libR"] in
+  let rmflg = C.Process.run c "pkg-config" ["--cflags"; "libRmath"] in
```

Comment:
Why do we need to make two separate calls for `libR` and `libRmath`?

Reply:
I split them up before I realizing the problem was with the trailing newline from `C.Process.run`.  Fixed.